db/migrations/20220509002323-create-github-app.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+module.exports = {
+	up: async (queryInterface, Sequelize) => {
+		return queryInterface.createTable('GithubApps', {
+			id: {
+				allowNull: false,
+				autoIncrement: true,
+				primaryKey: true,
+				type: Sequelize.INTEGER
+			},
+			githubBaseUrl: {
+				allowNull: false,
+				type: Sequelize.STRING
+			},
+			githubApiBaseUrl: {
+				allowNull: false,
+				type: Sequelize.STRING
+			},
```

Comment:
I don't think we need this as we can figure out the API url based on if it's a ghe instance or not